mix.exs

```diff
@@ -123,6 +128,54 @@ defmodule Sentry.Mixfile do
   end
 
   defp aliases do
-    [test: ["sentry.package_source_code", "test"]]
+    [
+      test: ["sentry.package_source_code", "test", &run_integration_tests_if_supported/1],
```

Comment:
Principle of least surprise: I think I would rather keep `mix test`'s behavior what we would expect it to be, which is run the repo's tests, and then have `mix test.integrations` separately (like you do now). This way users have to run it manually locally, but that's likely fine as this is more of a e2e test. CI will run it anyways, so it will catch broken stuff if that happens.

```suggestion
      test: ["sentry.package_source_code", "test"],
```